### DISCUSSION TITLE
Where is docs repo for adonisjs v5

### DISCUSSION THREAD

**Discussion by ehsanquddusi** [adonisjs/core#2142] [Help] [answered]
Hello,

Can someone direct me to correct `docs` repo/branch for Adonisjs v5. I can't find it in `adonisjs/docs` repo.

**Comment by thetutlage** [comment, marked as answer]
Here it is https://github.com/adonisjs/adonisjs.com inside the `docs` directory

**Reply by ehsanquddusi** [reply]
Thank you!
